content/2017/1/25/parallax.rst

```diff
@@ -0,0 +1,36 @@
+public: yes
+author: sondra
+tags: [design]
+image:
+  - src: '2017/parallax/parallax.jpg'
+summary: |
+  Whether you like it not, parallax scrolling is a reality, so let's learn how to do it well with CSS 3D. Paul Lewis' guide to performant parallaxing is the perfect place to start.
+
+
+Let's Talk About Parallax: Dos and Don'ts
```

Comment:
I like that! As a pattern, it could tend towards long titles, but that's probably ok. 

(We could eventually bake some of these patterns into the code, by adding templates for different post-types. Then we could get `<type>: <title>` in a more structured/designed way.)